samples/mongodb/simple/sample.md

```diff
@@ -10,54 +10,66 @@ redirects:
 - samples/mongodb-tekmaven/databus
 ---
 
-
 ## Prerequisites
 
-Ensure an instance of [MongoDB](https://www.mongodb.com/) is running on `localhost:27017`. See [Install MongoDB on Windows](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/).
+Ensure an instance of [MongoDB](https://www.mongodb.com/) is executing on `localhost:27017`.
+
+The easiest way to do this is to run MongoDB in Docker by running the following commands
+
+```shell
+docker run -d -p 27017:27017 --name TestMongoDB mongo:latest --replSet tr0
+```
 
+```shell
+docker exec -it TestMongoDB mongosh --eval 'rs.initiate()'
+```
 
-### MongoDB management UI
+Alternatively, it is possible to [install MongoDB](https://www.mongodb.com/docs/manual/installation/) but the instance must be part of a [replica set](https://www.mongodb.com/docs/manual/replication/) to enable transactions. If this is not possible, the endpoint configuration must be altered to disable transactions (not recommended for production):
 
-To visualize the data in MongoDB, it is useful to install a [MongoDB administration tool](https://docs.mongodb.com/ecosystem/tools/). The screen shots shown in this sample use [Robomongo](https://robomongo.org/).
+```c#
+endpointConfiguration.UsePersistence<MongoPersistence>().UseTransactions(false)
+```
```

Comment:
I'd offload this to a snippet.

Reply:
Yeah it might make sense to do that to make sure things continue to compile. Although it might look weird to have a class in the sample that is not used.